.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.2
```

Comment:
This leads to an error message for me:

```
rbenv: version `ruby-2.2' is not installed 
```

I think we should leave this out.

Reply:
Oki. But you need Ruby 2.2 to run this project?? I'll remove it though

Reply:
Yeah, I don't think there's a mechanism at the moment that says "You can build/run this project with any Ruby starting from version 2.2". In any case, if someone tries to build with 2.1 or something they'll find out soon enough :man_shrugging:.